with-sidecar-http-service/nuclio-func.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	nuclio "github.com/nuclio/nuclio-sdk-go"
+	"github.com/valyala/fasthttp"
+	"net/url"
+	"os"
+)
+
+func Handler(context *nuclio.Context, event nuclio.Event) (interface{}, error) {
+	context.Logger.Info("Got request, sending it to sidecar container")
+	req := fasthttp.AcquireRequest()
+	req.SetBody(event.GetBody())
+	sidecarHost := os.Getenv("SIDECAR_HOST")
+	sidecarPort := os.Getenv("SIDECAR_PORT")
+	sidecarUrl, _ := url.JoinPath(fmt.Sprintf("%s:%s", sidecarHost, sidecarPort), event.GetPath())
+	context.Logger.Info(sidecarUrl)
+	req.SetRequestURI(sidecarUrl)
+	resp := fasthttp.AcquireResponse()
+	err := fasthttp.Do(req, resp)
+	return nuclio.Response{
+		StatusCode:  resp.StatusCode(),
+		ContentType: "application/text",
+		Body:        resp.Body(),
+	}, err
+}
```

Comment:
Some new lines between sections in the code can help read the flow.

Reply:
+ add some comments. 
because it is a template and basically an example for users, we want it to be as clear as it gets